src/desktop/components/react/stitch_components/index.tsx

```diff
@@ -0,0 +1,12 @@
+export { ReactionArtwork as Artwork } from "./ReactionArtwork"
+export {
+  ReactionArtworkArtistInfo as ArtworkArtistInfo,
+} from "./ReactionArtworkArtistInfo"
+export {
+  ReactionArtworkDetails as ArtworkDetails,
+} from "./ReactionArtworkDetails"
+export { ReactionArtworkGrid as ArtworkGrid } from "./ReactionArtworkGrid"
+export { ReactionFillwidth as Fillwidth } from "./ReactionFillwidth"
+export {
+  ReactionTooltipQuestion as TooltipQuestion,
+} from "./ReactionTooltipQuestion"
```

Comment:
New additions that are exported from this index file should be available to stitch

Reply:
This Tooltip Question should eventually go away. I believe there is a new Palette Tooltip being added that should replace this one. Not really for this change at all.

Reply:
There are slight differences between old and new, can follow up once that's all ready. Either way this file would need to be updated somehow or deleted; this just organizes things.